src/modules/bigquery.js

```diff
@@ -64,25 +68,12 @@ export default class BigQuery {
         });
     });
 
-  calculateColumns = queryResults => {
-    const columns = [];
-    if (
-      queryResults.rows !== undefined &&
-      queryResults.schema !== undefined &&
-      queryResults.schema.fields !== undefined
-    ) {
-      queryResults.schema.fields.forEach(colData => {
-        const col = {
-          title: colData.name,
-          field: colData.name,
-        };
-
-        columns.push(col);
-      });
-    }
-    return columns;
+  commaFormatted = amount => {
+    return amount.replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
Let's use this built-in instead of a regex: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat

`return new Intl.NumberFormat('en-US', { maximumSignificantDigits: 3 }).format(amount);`

This function defaults `useGrouping` to true, which specifies "whether to use grouping separators, such as thousands separators or thousand/lakh/crore separators."